opencti-platform/opencti-graphql/src/domain/group.js

```diff
@@ -54,7 +54,12 @@ export const batchRoles = async (context, user, groupIds) => {
 };
 
 export const addGroup = async (context, user, group) => {
-  const created = await createEntity(context, user, group, ENTITY_TYPE_GROUP);
+  const groupWithDefaultValues = {
+    ...group,
+    default_assignation: false,
```

Comment:
⚠️ In this order, the default_assignation and auto_new_marking property will be overriden if they exist in the group object.

Reply:
owww right !!